app/views/users/index.html.erb

```diff
@@ -1 +1,16 @@
-<h1>Here is the list of users</h1>
\ No newline at end of file
+<section class="list-container">
+  <div class="user-list">
+    <% @users.each do |user| %>
+    <div class="user-container-index">
+      <div class="img-user">
+        <img src='<%= user.photo %>' alt='user-picture'>
+      </div>
+      <div class="user-info">
+        <p class="user-name">
+          <%= link_to user.name, user_path(id: user.id) %>
+        </p>
+        <p class="user-numbers">Number of Posts: <%= user.posts_counter || 0 %></p>
+      </div>
+  </div>
+  <% end %>
+</section>
```

Comment:
- [ ] It is not a problem if ı click on the first post 1 but if ı click on the post 3 of the page ı am getting an error. Please fix it.
![image](https://user-images.githubusercontent.com/38635158/173119488-f8f463ac-c595-4dfc-9536-126e58171f19.png)
![image](https://user-images.githubusercontent.com/38635158/173119579-58fbaaa5-7298-40f2-9f55-4e324f8ab3f1.png)

Reply:
Hello @ahmetbozaci , thank youf or the review.

I just created a new user with 3 posts and when I go in all of them I don't get that error, could you tell me how you got yours? 
![pic3](https://user-images.githubusercontent.com/92347711/173124360-a3e693e7-959c-4646-982f-f9ae93be6d32.png)
![index](https://user-images.githubusercontent.com/92347711/173124363-4c8c6fdb-efa3-4aa9-b9aa-59a021b3013b.png)
![pic1](https://user-images.githubusercontent.com/92347711/173124368-b87e1814-cdb0-4d1d-a87b-070f084bcc88.png)
![pic2](https://user-images.githubusercontent.com/92347711/173124373-661f241a-8571-4d2f-a80e-3f73d994d9db.png)

Reply:
@Thi-Ponce
**I create these samples and ı get error when click post 4 and post 3.  I dont know why. It has not problem when click post 1 and post 2**
 `first_user = User.create(name: 'Tom', 
  photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80', 
  bio: 'Teacher from Mexico.', posts_counter:0)
second_user = User.create(name: 'Lilly', 
  photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80', 
  bio: 'Teacher from Poland.', posts_counter:0)

  first_post = Post.create(author: first_user, title: 'Post 1', text: 'This is my first post',comments_counter:0, likes_counter:0)
  second_post = Post.create(author: second_user, title: 'Post 2', text: 'This is my second post',comments_counter:0, likes_counter:0)
  third_post = Post.create(author: first_user, title: 'Post 3', text: 'This is my third post',comments_counter:0, likes_counter:0)
  fourth_post = Post.create(author: second_user, title: 'Post 4', text: 'This is my fourth post',comments_counter:0, likes_counter:0)

  Comment.create(post: first_post, author: second_user, text: 'Hi Tom!' )
  Comment.create(post: second_post, author: first_user, text: 'Hi Tom!' )
  Comment.create(post: second_post, author: second_user, text: 'Hi Tom!' )
  Comment.create(post: third_post, author: first_user, text: 'Hi Tom!' )
  Comment.create(post: third_post, author: second_user, text: 'Hi Tom!' )
  Comment.create(post: third_post, author: second_user, text: 'Hi Tom!' )`

Reply:
Hey @ahmetbozaci , I just ran the commands you sent the screenshot and it work normally on my end, I really don't know what to do.

Reply:
> Hey @ahmetbozaci , I just ran the commands you sent the screenshot and it work normally on my end, I really don't know what to do.

@Thi-Ponce If everything works fine please send a review again. The next code reviewer can check.